sui-execution/Cargo.toml

```diff
@@ -45,3 +45,16 @@ workspace-hack.workspace = true
 [dev-dependencies]
 cargo_metadata = "0.15.4"
 petgraph = "0.5.1"
+
+[features]
+default = []
+gas-profiler = [
+    "sui-adapter-latest/gas-profiler",
+    "sui-adapter-v0/gas-profiler",
+    "sui-adapter-v1/gas-profiler",
+    "move-vm-runtime-next-vm/gas-profiler",
+    "move-vm-runtime-v0/gas-profiler",
+    "move-vm-runtime-v1/gas-profiler",
+    "move-vm-runtime-latest/gas-profiler",
+    "move-vm-config/gas-profiler",
```

Comment:
You will need to add template strings here as well, for `execution_layer.py` to pick up.  It uses these to update the TOML file when it adds a new cut.

```suggestion
    "sui-adapter-latest/gas-profiler",
    "sui-adapter-v0/gas-profiler",
    "sui-adapter-v1/gas-profiler",
#     "sui-adapter-$CUT/gas-profiler",
    "move-vm-runtime-next-vm/gas-profiler",
    "move-vm-runtime-v0/gas-profiler",
    "move-vm-runtime-v1/gas-profiler",
    "move-vm-runtime-latest/gas-profiler",
#     "move-vm-runtime-$CUT/gas-profiler",
    "move-vm-config/gas-profiler",
```

Reply:
thanks! where is this documented?

Reply:
This would be the best place to learn more:

https://github.com/MystenLabs/sui/blob/f9b1a57caaf9828e6b25f5c9ceb7a58984577f94/sui-execution/README.md

Followed by:

https://github.com/MystenLabs/sui/blob/f9b1a57caaf9828e6b25f5c9ceb7a58984577f94/sui-execution/cut/README.md